awesome_owl/static/src/card/card.xml

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<templates xml:space="preserve">
+
+    <t t-name="awesome_owl.card">
+        <div class="card d-inline-block m-2" style="width: 18rem;">
+            <div class="card-body">
+                <h5 class="card-title">
+                    <t t-esc="props.title"/>
```

Comment:
```suggestion
                    <t t-out="props.title"/>
```
t-esc is deprecated, you should have a warning in your logs that tells you to use `t-out` instead :wink: